BootloaderCorePkg/Stage2/Stage2Support.c

```diff
@@ -403,33 +404,26 @@ BuildBaseInfoHob (
     SerialPortInfo->UartPciAddr = 0;
   }
 
-  // Build Frame Buffer hob
-  FspGfxHob = (EFI_PEI_GRAPHICS_INFO_HOB *)GetGuidHobData (LdrGlobal->FspHobList, &Length,
+  // Build graphic info hob
+  BlGfxHob = BuildGuidHob (&gEfiGraphicsInfoHobGuid, sizeof (EFI_PEI_GRAPHICS_INFO_HOB));
+  ZeroMem (BlGfxHob, sizeof (EFI_PEI_GRAPHICS_INFO_HOB));
```

Comment:
ZeroMem needs to be inside if (blGfxHob != NULL)

Reply:
thank to capture this. will updated in next update.